### PR TITLE
Add foreign key definition and example

### PR DESCRIPTION
**Pull request by sirhung1993**
A very basic foreign key definition and example

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by sirhung1993** [comment]
I cannot find the button to select as this sentence: 
"Make sure the “Create a new branch for this commit and start a pull request” radio button is selected".
Please guide me more if I did it wrong.

File: guide/english/computer-science/databases/relational-databases/index.md [hunk 1]
```diff
@@ -61,6 +61,37 @@ Figure 4 - Example of Pet table with owner field linked.
 
 A one-to-many relationship is one record linked to many other records, the example being the user Jerry having two pets. It could also be a many-to-many relationship where the tables could be books and authors, as authors could co-write many books. Finally the most common relationship type is one-to-one, a record that can only be linked to one, and only one, other record.
 
+### Foreign key
+
+To describe the Relationships above, Foreign keys(FK) should be be used to link table to tables.
```
Comment (nikrb) [inline guide/english/computer-science/databases/relational-databases/index.md]: typo: ... should be **be** used ...
I think the end would be better as: `... used to link tables.`

**Review by nikrb — DISMISSED** [review]
small typo and grammar

File: guide/english/computer-science/databases/relational-databases/index.md (hunk 1 above)
Comment (sirhung1993) [reply guide/english/computer-science/databases/relational-databases/index.md]: Thanks @nikrb!
I have update as your recommendation.

**Review by cmccormack — APPROVED** [review]
Looks good to me.  Please don't use abbreviations where not necessary.  Unless it is very common to refer to it as FK, just use `foreign key`

**Pull request merged**